Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by Dasan on 2023/06/02.
+//
+
+import Foundation
+
+enum ExpressionParser {
+    static func parser(from input: String) -> Formula {
+        var formula = Formula(operands: CalculatorItemQueue(),
+                              operators: CalculatorItemQueue())
+        
+        let operands = componentsByOperators(from: input).compactMap { Double($0) }
+        
+        operands.forEach {
+            formula.operands.enqueue($0)
+        }
+    
+        let operators = input.filter({ $0.isNumber == false }).compactMap({ Operator(rawValue: $0) })
```

Comment:
후행클로저는 `()`를 생략하는게 좋을 것 같아요.
또한, 개인적인 취향인데 고차함수별로 개행을 주는게 가독성에 좋은 것 같아요.
```suggestion
        let operators = input.filter { $0.isNumber == false }
                             .compactMap { Operator(rawValue: $0) }
```

Reply:
후행 클로저의 `()` 삭제 및 개행을 완료하였습니다:)